server/controllers/importers/graphml.js

```diff
@@ -2,7 +2,7 @@ var constants = require(__dirname + "/../constants");
 var parseString = require("xml2js").parseString;
 var semanticChecker = require(__dirname + "/../semantic-checker");
 
-/* 
+/*
```

Comment:
Ah OK…it wasn’t apparent that this was something he was going to come back to. Let’s try to establish a convention that when we are commenting out code rather than deleting it, we explain why. (like you did with the #451-related commented-out tests)